src/averageAges.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+function avarageAge(people) {
+  const agesPeople = people.map(person => person.died - person.born);
+  const acc = 0;
+
+  return agesPeople.reduce((prev, current) => prev + current, acc)
+   / agesPeople.length;
```

Comment:
Let's combine this methods, we can calculate age in `reduce` method, why to iterate methods twice 🙂